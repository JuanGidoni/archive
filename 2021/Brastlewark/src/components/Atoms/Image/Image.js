const Image = ({ src, alt }) => {
 return (
  <img src={src} alt={alt} className="w-100 h-auto" style={{ objectFit: 'cover' }} />
 );
};

export default Image;
