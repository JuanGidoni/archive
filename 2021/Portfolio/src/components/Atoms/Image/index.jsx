import './index.css';

const Image = () => {
 return (
  <>

  </>
 )
}

export default Image
