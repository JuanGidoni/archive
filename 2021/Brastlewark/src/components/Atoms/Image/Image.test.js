import React from 'react';
import renderer from 'react-test-renderer';
import Image from './Image';

describe('<Image />', () => {
 it('renders correctly and match snapshot', () => {
  const tree = renderer
   .create(<Image src="https://example.com/example.png" alt="Example" className="example" />)
   .toJSON();
  expect(tree).toMatchSnapshot();
 });
});