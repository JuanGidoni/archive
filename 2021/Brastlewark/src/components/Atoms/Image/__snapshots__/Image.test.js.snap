// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Image /> renders correctly and match snapshot 1`] = `
<img
  alt="Example"
  className="w-100 h-auto"
  src="https://example.com/example.png"
  style={
    Object {
      "objectFit": "cover",
    }
  }
/>
`;
